views/tables/StudiesTable/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import dayjs from 'dayjs'
+
+import { SITE_NAMES } from '../../../server/utils/siteNames'
+import StatusChip from '../../components/StatusChip'
+import Table from '../Table'
+
+const StudiesTable = (props) => {
+  const { onSort, studies, sortable, sortDirection, sortProperty } = props
+
+  const headers = [
+    {
+      dataProperty: 'study',
+      label: 'Study',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'site',
+      label: 'Site',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'daysInStudy',
+      label: 'Days In Study',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'numOfParticipants',
+      label: 'No. of Participants',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'updatedAt',
+      label: 'Last Updated',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'status',
+      label: 'Status',
+      sortable: false,
+    },
+  ]
+
+  const cellRenderer = (study, property) => {
+    switch (property) {
+      case 'updatedAt': {
+        return dayjs(study[property]).toISOString()
+      }
+      case 'site': {
+        return SITE_NAMES[study['study']]
+      }
+      case 'status': {
+        const isActive = true
```

Comment:
why do we need this constant if it is always true?

Reply:
The client hasn't communicated how we determine if a site is active or inactive. 
https://gnardog.slack.com/archives/C03KGHSSTNC/p1712151244130239

So for now.. they're all active

Reply:
ok so then lets either remove the status column or remove this constant and change `<StatusChip isActive={isActive} />` to `<StatusChip isActive />`. We can add the logic or column back when we know more.

Reply:
[112650b](https://github.com/AMP-SCZ/dpdash/pull/723/commits/112650bca788220e00e45c0c6cca236a02c75414)